doc/man3/SSL_CTX_set_tlsext_ticket_key_cb.pod

```diff
@@ -126,9 +126,9 @@ The I<hctx> key material can be set using L<HMAC_Init_ex(3)>.
 
 =head1 NOTES
 
-Session resumption shortcuts the TLS so that the client certificate
-negotiation don't occur. It makes up for this by storing client certificate
-an all other negotiated state information encrypted within the ticket. In a
+Session resumption shortcuts the TLS handshake so that the client certificate
```

Comment:
It's unclear why this talks about a client certificate

Reply:
Because it is one of the parameters of the original session that the server could be interested in?

Reply:
The manpage talks about clients using this function.

Reply:
I guess I should read better